### DISCUSSION TITLE
Loan interests too small to have an impact

### DISCUSSION THREAD

**Discussion by victorleblais** [voxeltycoon/issues#460] [General]
From this discussion: https://steamcommunity.com/app/732050/discussions/0/4679778856503365677/

The loan in the game has a very low interest rate, which makes it basically useless. Even on the hardest setting, the loan is still basically free money.

The interest rate could be increased across the board.

Not sure how it would feel, but something like 15-20% on hardest and 5% on easiest?

**Comment by JezzyBelle** [comment]
I agree generally. I also think that there needs to be a class of loans based upon timing and credit rating. 
Start Up Loan. The initial loan would be no-interest and is handled separately. You cannot increase the loan. Once you pay that amount back it is gone. 
When faced with a cash or expansion issue you now have the following options.
See your assets..
Get a Secured Loan, Take a loan against certain assets. Default, and poof, they are gone. Better short-term option for many than just selling assets if executed wisely. 
Unsecured Loan. Based upon credit worthiness and simple history. A high Debt as opposed to Cash and asset ratio gives you a higher interest rate. Beg for cash when you are broke too many times and they may tell you to hit the road or offer to work with you on a secured loan. YOu can have multiple loans. Each one paid off helps with the rating as well.

**Comment by ellielo** [comment]
Short term loans already have pretty high rates from experience. Had to use twice : one time personal loan and one time payday loan from [noeltichy](https://noeltichy.com/m/bad-credit-loans/). I think  I found the best deal on the market at that time. It saved the situation, I paid both on time, everyone is happy

**Comment by WoosterUK** [comment]
The other thing the bank (aka devs) should do is ramp up the interest rate for a new loan based on a comparison between outstanding debt and gross assets. Ideally, this would be built in following improved accounting standards (see https://github.com/voxeltycoon/issues/discussions/799), so that debt could be compared against total assets valued properly (i.e., including depreciation for vehicles). A dynamic interest rate would provide a more natural brake on growth.
